packages/nimble-components/src/table-column/base/tests/table-column.fixtures.ts

```diff
@@ -58,3 +60,59 @@ export class TableColumnDelegatesClickAndKeydown extends TableColumn {
         });
     }
 }
+
```

Comment:
The intention of this file is not to place all table column fixtures but for shared column fixtures used across multiple tests. I think that was overlooked for the TableColumnDelegatesClickAndKeydown fixture and TableColumnValidationTest fixture below.

These are very test specific and don't seem generally useful to declare once globally. They should be moved to their own files and follow the normal pattern for defining components specific to a test

Reply:
I am using this test column type from two different test files, though: `table.spec.ts` and `table-validator.spec.ts`, so it seems better to put it in `table-column.fixtures.ts` than to duplicate the code. (I'll move `TableColumnDelegatesClickAndKeydown`.)

Reply:
Ah got my wires crossed because some tests were constructing with the constructor vs the tagName. Your described direction makes sense 👍